spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -190,45 +195,51 @@ public void shutdown() throws Exception {
 		}
 	}
 
+	/**
+	 * Will check if {@link LocalDeployerProperties#INHERIT_LOGGING} is set by
+	 * checking deployment properties and then application properties
+	 */
+	private boolean shouldInheritLogging(AppDeploymentRequest request){
+		boolean logToConsole = false;
+		if (request.getDeploymentProperties().containsKey(LocalDeployerProperties.INHERIT_LOGGING)){
+			logToConsole = Boolean.parseBoolean(request.getDeploymentProperties().get(LocalDeployerProperties.INHERIT_LOGGING));
+		}
+		else if (request.getDefinition().getProperties().containsKey(LocalDeployerProperties.INHERIT_LOGGING)){
```

Comment:
Actually it is not deployer only concern. The idea is to have two types of control
1. Per deployer
2. Per application
So think of SCDF flow that has bunch of applications and you only care about redirecting logging for one?. . . or you may want all. In the first case it is 'definition' property and the later it is 'deployer'. 
Does it clarify or am I missing something?